go/libkb/pgp_key.go

```diff
@@ -607,6 +609,13 @@ func (k *PGPKeyBundle) CheckFingerprint(fp *PGPFingerprint) error {
 }
 
 func (k *PGPKeyBundle) SignToString(msg []byte) (sig string, id keybase1.SigID, err error) {
+	if k.Entity.PrivateKey == nil || k.Entity.PrivateKey.PrivateKey == nil {
```

Comment:
So say somewhere in the code we have a `PGPKeyBundle` made from a public key.  And then say the code doesn't check `CanSign()` and calls `SignToString()`.  `k.Entity.PrivateKey` will be nil and this code will be executed and might succeed.

I'm just worried that someone who explicitly chooses not to have keybase import their private key could get on a path where keybase (via gpg) successfully signs something with that key and they freak out.